### PR TITLE
Adds posts#show view template that can be overriden

### LINKED ISSUE #76: Inline rendering of blog posts prevents customization of the view for Posts#show
### Expected behavior
To be able to customize the presentation of a single blog post by changing the view template.

### Actual behavior
Blog posts' `#show` action is using inline rendering which displays the content but there's no access to the additional fields - e.g. `title`, `published_at`, etc, unless they are added a second time in the template itself.

The line responsible for this is https://github.com/comfy/comfy-blog/blob/7f04203cb5adbda452e5acc3c2df68cb65bdb6b9/app/controllers/comfy/blog/posts_controller.rb#L25 

If this is changed to simply

````ruby
render  layout: app_layout
````

And a simple view template is added in `app/views/comfy/blog/posts/show.html.erb` it will allow full customization when viewing a single blog post.

### System configuration
**Rails version**:
5.2.0

**CMS version**:
2.0.15

**Ruby version**:
2.5.1p57

### PR DESCRIPTION
**Pull request by kirichkov**
### Summary

Closes #76

### CODE REVIEW

File: test/controllers/comfy/blog/posts_controller_test.rb [hunk 1]
```diff
@@ -75,10 +75,19 @@ def test_get_index_with_category_invalid
...
 
   def test_get_show
+    expected_content =
+      "<h1>Default Title</h1>\n"\
+      "<p>\n"\
+      "Published on\n"\
+      ":\n"\
+      "#{@post.published_at.to_formatted_s(:short)}\n"\
+      "</p>\n"\
+      "blog post content\n"\
```
Comment (GBH) [inline test/controllers/comfy/blog/posts_controller_test.rb]: I'd change this to `expected = <<~HTML` heredoc format.

**Comment by GBH** [comment]
Makes sense to me. Adjust the test and let's merge this.

File: test/controllers/comfy/blog/posts_controller_test.rb (hunk 1 above)
Comment (kirichkov) [reply test/controllers/comfy/blog/posts_controller_test.rb]: Fixture is also dynamic - `2.days.ago` so either I change the fixture, or keep it dynamic in the expected string.

Comment (kirichkov) [reply test/controllers/comfy/blog/posts_controller_test.rb]: It appears `#published_at` is not used anywhere else, so I changed it to a fixed value

**Pull request merged**